auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -359,6 +368,7 @@ export default class AccountInfo extends Mixins(
   private originalAddress: Address = null // store the original address..do not modify it afterwards
 
   private errorMessage: string = ''
+  private warningMessage: string = ''
```

Comment:
convert to composition-api?

Reply:
Need to create new ticket for it

Reply:
ok another PR? Would be nice..  we have 74~ components converted and another 140~ to go